cmd/nuclei/main.go

```
@@ -215,6 +221,10 @@
 	}
 }
 
+var (
+	nameGenerator = namegenerator.NewNameGenerator(time.Now().UnixNano())
```

golangci-lint (1.62.2)

var `nameGenerator` is unused

(unused)